cloudhub-2/modules/ROOT/pages/ps-config-app-level-egress.adoc

```diff
@@ -143,8 +143,12 @@ To add application-level egress support to API Manager, add a custom rule group
 *** For prod-eu: `eu1.anypoint.mulesoft.com`
 
 [NOTE]
+====
 When configuring application-level egress connections to Anypoint MQ, Object Store v2, or API Manager, you must also allow outbound access to all 443 ports at the private-space level separately. For more information about allowing outbound access to 443 ports, visit xref:ps-config-fw-rules.adoc[]. 
 
+You can't use custom domains private with space internal DNS.
```

Comment:
This sentence is unclear. I think it should be "You can't use custom private domains...", but I'm not sure how to revise the rest of the sentence.